mautic/mautic
# Unsubscribe form fixed

Reported in https://github.com/mautic/mautic/issues/702. There were 2 unset variables which has to be taken care of.

#### Test
1. Create a form. Call it Unsubscribe form for example. One text field will do the trick.
2. Create an email. Call it Unsubscribe test for example. Assign the Unsubscribe form as unsubscribe form.
3. Add the unsubscribe link to the email body.
4. Send the form to yourself.
5. Click the link.

Before PR the Unsubscribe form throws a 500 error. After PR it displays the form.

It should work with all themes. I tested it with Nature. You can choose another one for your test.
